documentation/user_kit.md

```diff
@@ -198,6 +201,14 @@ Partners can invite users, without the need of creating them ahead of time, and
 	}
 	```
 
+   Validation
```

Comment:
Should we also explain the significance of `role` in the request since it is a new field?

Reply:
great call! I added information here.